MapboxNavigation/NavigationViewController.swift

```diff
@@ -382,10 +392,6 @@ public class NavigationViewController: NavigationPulleyViewController, RouteMapV
         return navigationDelegate?.navigationMapView?(mapView, routeCasingStyleLayerWithIdentifier: identifier, source: source)
     }
     
-    func navigationMapView(_ mapView: NavigationMapView, routeStyleLayerWithIdentifier identifier: String, source: MGLSource) -> MGLStyleLayer? {
-        return navigationDelegate?.navigationMapView?(mapView, routeStyleLayerWithIdentifier: identifier, source: source)
```

Comment:
Per https://github.com/mapbox/mapbox-navigation-ios/pull/270#discussion_r132099394, this PR still removes the ability to customize the route line.